### DISCUSSION TITLE
Can Factory Reset  leave any previous congifurations?

### DISCUSSION THREAD

**Discussion by lamprey13** [darthcloud/BlueRetro#1005] [Q&A]
So, I have a Genesis receiver. It came with v1.9 which seemed to be working just fine with my 8bitdo DIY kit.. I updated to 24.04. After much fiddeling with mappings and settings to get it working the way I wanted I was unable to. So, I installed 1.9.2. Again the mappings are off. Fixable, but off. So I did a factory reset to 1.9 and the mappings are still off. 

Is it possible that some configuration is "stuck" even after a factory reset?

I'm able to re-map the buttons to get around it, but I don't seem to be having the same issue on a different receiver...?

**Comment by lamprey13** [comment]
Maybe this is a hardware issue.

I have two receivers I've been comparing: a Stand-alone Receiver and a Core Receiver with genesis break out cables.

They are both setup with:

- v1.9.1 FW 
- Global Config - System set to MD-Genesis 
- Output Config - Mode set to Game Pad Alt

I haven't made any other changes (other than those settings above) after doing a factory rest and installing v1.9.1

The Stand-alone Reciver that has mappings wrong. The Core Receiver has the mappings correct.

Is there anyway to diagnose the stand-alone receiver, other then my observations that the mappings are incorrect? and/or is there a place to buy a stande-alone receiver that are known to be good?

**Comment by darthcloud** [comment]
If you update to v25.01 you can download a debug trace. And I see if something is wrong in mapping.:
https://github.com/darthcloud/BlueRetro/wiki/Debug-trace
